### DISCUSSION TITLE
Drop Indicator Without Moving Draggable Source

### DISCUSSION THREAD

**Discussion by BrianHung** [clauderic/dnd-kit#789] [Q&A] [answered]
Is there a way to show a drop indicator _without_ moving the draggable source?

Good example is in Figma where they should you a black bar representing the drop indicator but keeping the source in palce.

![CleanShot 2022-06-08 at 17 13 33](https://user-images.githubusercontent.com/15721634/172738069-5e71d758-9e07-48d4-bfbb-7cb0765abe21.gif)

**Comment by clauderic** [comment, marked as answer]
Yes, refer to these two examples:

https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/story/examples-pages-layout--grid
https://github.com/clauderic/dnd-kit/tree/master/stories/3%20-%20Examples/Advanced/Pages

https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/story/examples-tree-sortable--drop-indicator
https://github.com/clauderic/dnd-kit/tree/master/stories/3%20-%20Examples/Tree

If you don't want to move the draggable source at all, don't pass it the `transform` property and don't use `<DragOverlay>`.

**Reply by BrianHung** [reply]
Trying to modify this

https://github.com/clauderic/dnd-kit/blob/17ab9cee2479115ffeb1df67057ebcfd0f1fca6e/stories/3%20-%20Examples/Advanced/Pages/Pages.tsx#L190-L196

to use instead the pointer position and the hovered over rectangle midpoint

```
   over?.id === id 
     ? pointerY > (rect.top + rect/bottom) / 2 // midpoint
       ? Position.After 
       : Position.Before 
     : undefined 
```

Is there a way to access the current pointer position relative to the same rectangles as `over.rect`?
